src/pages/cli/graphql/examples-and-solutions.mdx

```diff
@@ -564,4 +564,83 @@ query repsByPeriodAndTotal {
     }
   }
 }
-```
\ No newline at end of file
+```
+
+## Advanced
+
+### Batch Put Custom Resolver
+
+Sometimes it's preferred to create objects in bulk, rather than subsequently creating individual objects and waiting for all the requests to complete.
```

Comment:
```suggestion
Sometimes you need to create objects in bulk, rather than creating individual objects sequentially and waiting for all the requests to complete.
```